src/DynamoRevit/DynamoRevit.cs

```diff
@@ -1,3 +1,4 @@
+using System.Collections.Generic;
```

Comment:
Hi @sharadkjaiswal, I have tried to work on this locally but I do not have `Revit 2016` installed, so the code I write might be error-prone (it cannot be built). Since that is the case, I'll suggest code changes directly from here.